README.md

```diff
@@ -17,15 +17,15 @@
 
 | Key | Function |  
 |:--------------|:----------------:|
-| SPACE | Toggle pause / play |
+| SPACE | Pause or play |
 | N | Play next track |
 | R | Replay current track |
 | F | Toggle fullscreen |
 | L | Play last played track / Track on the end of the list |
-| UP | Volume Up |
-| DOWN | Volume Down |
-| LEFT | Track Seek -10s |
-| RIGHT | Track Seek +10s |
+| UP | Volume up in 10% increments |
+| DOWN | Volume down in 10% increments |
```

Comment:
Volume down in 10% increments ? makes sense

Reply:
Means volume goes up (eg: from 10 to 20) up button. Same for the down

Reply:
Better if we make it
```md
| UP | Increment in volume by 10% |
| DOWN | Decrement in volume by 10% |
```

Reply:
^ Perfect

Reply:
That 'in' in both lines is also not needed maybe ?

Reply:
@aashutoshrathi: Better than mine! Great! 👍

Reply:
@Anshuman-Verma Do you mean ?
Better if we make it
```md
| UP | Increments volume by 10% |
| DOWN | Decrements volume by 10% |
```

Reply:
Make it "Increments/Decrements"

Reply:
GFI @vanguard07